### PR TITLE
add : added input and text-utility component

### PR DESCRIPTION
**Pull request by JuvitaSaini09**
1.  added input component and text utility component 
2. added documentation of  input component and text utility component 

### Input component snap shot
image

### Text utility component snap shot
image

### CODE REVIEW

File: components/input-textUt/text-ut.html [hunk 1]
```diff
@@ -0,0 +1,99 @@
...
+        </div>
+
+
+        <h2 class="grey mb2">Small text, gray text and center text</h2>
+
+        <div class="typography-container">
+            <p>This is a paragraph</p>
+            <p class="grey-text">This is a grey text</p>
+            <small>This is small text</small>
+            <center>This is centered text</center>
```
Comment (gauravsinhaweb) [inline components/input-textUt/text-ut.html]: this is not the right method to do using HTML tags to small or center text, rather you should give a CSS class. How the user is supposed to access it!

File: components/input-textUt/text-ut.html [hunk 2]
```diff
@@ -0,0 +1,99 @@
...
+            <li><a href="/components/html/toast.html">Toast/Snackbar</a></li>
+            <li><a href="/components/html/card.html">Simplified Grid</a></li>
+        </ul>
+    </div>
+
+    <div class="Component-container">
+
+        <!---------Text-utilities componenet-------->
+        <div class="text pb2 " id="text-ut">
+            <h2 class="pb1 h2 comp-title">Text utilities</h2><br>
```
Comment (gauravsinhaweb) [inline components/input-textUt/text-ut.html:52]: remove br tag if there is no use

File: components/input-textUt/text-ut.css [hunk 3]
```diff
@@ -0,0 +1,16 @@
+.grey-text {
+    color: grey;
+}
+
+.typography-container>* {
+    padding-bottom: 2rem;
+}
+
+.typography-container {
+    width: 90%;
```
Comment (gauravsinhaweb) [inline components/input-textUt/text-ut.css:17]: use rem value for dimensions i.e. width and height. Apply this for all.

File: components/input-textUt/text-ut.css [hunk 4]
```diff
@@ -0,0 +1,16 @@
+.grey-text {
+    color: grey;
```
Comment (gauravsinhaweb) [inline components/input-textUt/text-ut.css]: use color variables here.

**Review by gauravsinhaweb — CHANGES_REQUESTED** [review]

File: components/input-textUt/text-ut.html (hunk 1 above)
Comment (JuvitaSaini09) [reply components/input-textUt/text-ut.html]: I have made the changes,thankyou for review

File: components/input-textUt/text-ut.css (hunk 4 above)
Comment (JuvitaSaini09) [reply components/input-textUt/text-ut.css]: I have changed it, thanks for the review

File: components/input-textUt/input.css [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+    padding: 10px;
+    background-color:var(--white-light);
+    border: 1px solid var(--grey);
+    padding: 25px;
+    width: 60%;
+
+}
+
+.text-input {
+    font-size: 16px;
```
Comment (sri-rishi) [inline components/input-textUt/input.css]: use rem instead of px it will help in responsive

**Review by sri-rishi — CHANGES_REQUESTED** [review]
Except these changes code and UI looks good to merge

File: components/input-textUt/input.html [hunk 6]
```diff
@@ -0,0 +1,105 @@
...
+        </div>
+
+
+        <h2 class="grey mb2">Simple text input</h2>
+        <div class="text-input-container">
+            <label for="text-input">Name</label>
+            <input type="text" class="text-input" placeholder="Name">
+        </div>
+        <div>
+            <iframe id="frame"
```
Comment (manish-GH) [inline components/input-textUt/input.html:66]: Don't use Id unnecessarily if you are not using event handlers. Also, the same id cannot be used on multiple components. In your code every iframe has id='frame'. You should use class in this case.

Also avoid inline styling for iframes

**Review by gauravsinhaweb — APPROVED** [review]
Looks good to me. Nice work!

**Review by gauravsinhaweb — APPROVED** [review]

File: components/input-textUt/input.css (hunk 5 above)
Comment (JuvitaSaini09) [reply components/input-textUt/input.css]: i have made the change,thankyou for the review .please approove my PR

File: components/input-textUt/text-ut.css [hunk 7]
```diff
@@ -0,0 +1,29 @@
...
+
+.typography-container>* {
+    padding-bottom: 2rem;
+}
+
+.typography-container {
+    width: 90%;
+    border: 1px solid var(--light-grey);
+    border: 1px solid var(--grey);
+    padding: 25px;
```
Comment (manish-GH) [inline components/input-textUt/text-ut.css:20]: use rem

File: components/input-textUt/text-ut.css [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+    width: 90%;
+    background-color:var(--white-light);
+}
+.small-text{
+   font-size:12px;
```
Comment (manish-GH) [inline components/input-textUt/text-ut.css:25]: use rem. If you have used any value greater than 10px replace it with rem

**Review by sri-rishi — APPROVED** [review]
You can merge now

File: components/input-textUt/input.css (hunk 5 above)
Comment (manish-GH) [inline components/input-textUt/input.css]: use rem

File: components/input-textUt/input.css [hunk 9]
```diff
@@ -0,0 +1,44 @@
...
+    --green: rgb(19, 197, 19);
+    --red:red;
+    --orange:rgb(197, 128, 0);
+}
+.text-input-container {
+    width: 20rem;
+    padding: 10px;
+    background-color:var(--white-light);
+    border: 1px solid var(--grey);
+    padding: 25px;
```
Comment (manish-GH) [inline components/input-textUt/input.css:13]: use rem

**Review by manish-GH — APPROVED** [review]
Next time please do share the live version with the PR

File: components/input-textUt/input.html (hunk 6 above)
Comment (JuvitaSaini09) [reply components/input-textUt/input.html:66]: ok,i will take care of it,thnks for letting me know

**Pull request merged**